doc/release-notes/rl-2505.section.md

```diff
@@ -4,6 +4,13 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- `services.rippled` has been removed, as `rippled` was broken and had not been updated since 2022.
+
+- `services.rippleDataApi` has been removed, as `ripple-data-api` was broken and had not been updated since 2022.
+
```

Comment:
Should these be in a separate commit? They look not about Squid per se?

Reply:
This appears to be an artifact of the rebase on `main`, the release notes had a merge conflict beforehand. I can try to remove it

Reply:
Oh, it's my bad expecting the diff to be displayed reasonably then.